pages.pt_BR/common/bup.md

```diff
@@ -0,0 +1,24 @@
+# bup
+
+> Sistema de backup baseado no formato Git packfile, oferecendo salvamentos incrementais e desduplicação global.
+> Mais informações: <https://github.com/bup/bup>.
+
+- Inicializa um repositório de backup no diretório local especificado:
+
+`bup -d {{caminho/para/repositório}} init`
```

Comment:
The following example fix can be applied to the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/bup.md) page and then here:

```suggestion
- Inicializa um repositório de backup no [d]iretório local especificado:

`bup -d {{caminho/para/repositório}} init`
```